polaris-react/.storybook/GridOverlay/GridOverlay.tsx

```diff
@@ -18,12 +18,12 @@ interface Props {
 }
 
 export function GridOverlay({inFrame, maxWidth, layer, children}: Props) {
-  const [columns, setColumns] = useState(
-    window.innerWidth < BREAKPOINT ? COLUMNS_SMALL : COLUMNS_LARGE,
-  );
+  const {lgUp} = useBreakpoints();
+  const [columns, setColumns] = useState(lgUp ? COLUMNS_LARGE : COLUMNS_SMALL);
+
+  const handleResize = () => setColumns(lgUp ? COLUMNS_LARGE : COLUMNS_SMALL);
```

Comment:
@aaronccasanova this doesn't seem to be working as expected. I might be able to look at this later this aft, or ship as is and fix later. Let me know if you have any ideas

![image](https://screenshot.click/13-36-brbsj-7igsh.png)